### ISSUE TITLE
WAV file selection "fast scroll" alphabet slider for touch screen

### ISSUE THREAD

**Issue by centertrim** [FrSkyRC/ETHOS-Feedback-Community#255]
If you have hundreds of sound files on the radio (such as the amber sound pack and more), scrolling through the long special function track selection list to select the file you're interested in is a bit tedious.  It would be nice if the touchscreen radios included a 1-A-Z touch sensitive area on the left or right side of the wav file list that allows the user to touch in a region and jump to that relative position in the file list.  Then it would only take a little bit of scrolling from there in the main track list to find the file you are interested in.

Like this:  https://stackoverflow.com/questions/8192683/how-to-create-an-alphabetical-scrollbar-displaying-all-the-letter-in-android

**Comment by bactisme** [comment]
Another suggestion could be a search field for searching and filtering the list.

**Comment by mawzthefinn** [comment]
This could be nice. The selection experience is OK, but it drags when you have a large number of WAV files.

**Comment by jasc76** [comment]
Yes, please make a filtering or something to faster access sound files. <B
